### ISSUE TITLE
Vacation messages parsed by Perl, but not by Ruby

### ISSUE THREAD

**Issue by rdeavila** [sisimai/rb-Sisimai#65]
I'm receiving vacation messages in my server, and the Sisimai Ruby implementation (version 4.20.1) does not recognize as this. Running `Sisimai.make` on this messages returns `nil`.

But, if I run Sisimai using the `ytnobody/sisimai-api` Docker image (which uses the [Perl implementation](https://github.com/ytnobody/docker-sisimai-api) via `cpm install Sisimai`) the message is detected as `vacation` correctly.

```json
{
  "rows": [
    {
      "listid": "",
      "deliverystatus": "",
      "recipient": "[redacted]@[redacted].com.br",
      "replycode": "",
      "feedbacktype": "",
      "alias": "",
      "action": "",
      "token": "2b1a723ee283c1c4a6cf34734d223ba4c5927e06",
      "addresser": "[redacted]@[redacted].com.br",
      "rhost": "",
      "reason": "vacation",
      "timestamp": 1486754116,
      "smtpagent": "RFC3834",
      "messageid": "",
      "senderdomain": "[redacted].com.br",
      "subject": "",
      "destination": "[redacted].com.br",
      "smtpcommand": "",
      "lhost": "",
      "softbounce": -1,
      "diagnostictype": "",
      "timezoneoffset": "-0200",
      "diagnosticcode": "Estarei ausente do dia 08/02 atÃ© o dia 12/03 favor encaminhar email para [redacted] e [redacted]"
    }
  ],
  "status": 200
}
```

**Comment by azumakuniyuki** [comment]
Thanks for the bug report. Would you send (or attach to this issue) an email which you have received as a vacation message after replace secret information (such as email addresses, hostnames, and so on) with some string to us ?

We'll try to fix the bug on Ruby version of Sisimai.

Best regards.

**Comment by rdeavila** [comment]
That's the file: [jtfh15lics7ofhu91no9mmodi5o1dc09ql1t3ro1.eml.zip](https://github.com/sisimai/rb-Sisimai/files/777525/jtfh15lics7ofhu91no9mmodi5o1dc09ql1t3ro1.eml.zip).

Thank you.

**Comment by azumakuniyuki** [comment]
Thanks for the file.
We've found the bug at lib/sisimai/message/email.rb for matching the following line:

```email
To: 
	bounces+[redacted]@[redacted].net.br
```

The same regular expressions of `header` method for capturing each email header don't perform the same operation in the Perl version of Sisimai and the Ruby version of Sisimai.
We'll try to inspect and fix the bug.

**Comment by azumakuniyuki** [comment]
```perl
        if( $e =~ m/\A([^ ]+?)[:][ ]*(.+?)\z/ ) {
```
Code above can capture `To: ` line,
https://github.com/sisimai/p5-Sisimai/blob/master/lib/Sisimai/Message/Email.pm#L212

```ruby
          if cv = e.match(/\A([^ ]+?)[:][ ]*(.+?)\z/)
```
but the same regular expression in Ruby version of Sisimai does not capture `To: ` line.
https://github.com/sisimai/rb-Sisimai/blob/master/lib/sisimai/message/email.rb#L226

**Comment by azumakuniyuki** [comment]
Fixed the issue at pull-request #66. And we've registered new sample email `rfc3834-06.eml` (it based on the email you uploaded) into `set-of-emails/maildir/bsd` directory.

Please try to master branch of this repository.

Best regards,

**Comment by rdeavila** [comment]
Hi @azumakuniyuki 

The `master` branch is working! Thank you so much for your support.
